### PR TITLE
Keyboard layout skins fix 13.col visible need backport

### PR DESCRIPTION
**Pull request by Wolftronics-SBC**
Hungarian.xml first row 13 characters
## Types of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves existing functionality)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the [Code guidelines](https://codedocs.xyz/xbmc/xbmc/code_guidelines.html) of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [ ] I have read the [CONTRIBUTING](https://github.com/xbmc/xbmc/blob/master/CONTRIBUTING.md) document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

**Comment by ronie** [comment]
thx! changes look good to me.

i'll take care of the additional work needed to make the extra button fit within the layout.

**Comment by Wolftronics-SBC** [comment]
You're welcome. next time I try harder. i thank you too!

File: addons/skin.estuary/xml/DialogButtonMenu.xml [hunk 1]
```diff
@@ -30,6 +30,7 @@
 					<item>
 						<label>$LOCALIZE[20150]</label>
 						<onclick>AlarmClock(shutdowntimer,Shutdown())</onclick>
+						<visible>false</visible>
```
Comment (ronie) [inline addons/skin.estuary/xml/DialogButtonMenu.xml]: some unrelated change, please remove this.

File: system/keyboardlayouts/hungarian.xml [hunk 2]
```diff
@@ -7,15 +7,15 @@ Default font lacks support for all characters
   <layout language="Hungarian" layout="QWERTZ">
     <keyboard>
       <row>0123456789öüó</row>
-      <row>qwertzuiopőú</row>
-      <row>asdfghjkléáű</row>
-      <row>íyxcvbnm,.-</row>
+      <row>qwertzuiopőú+</row>
+      <row>asdfghjkléáű-</row>
+      <row>íyxcvbnm,.?!@</row>
```
Comment (ronie) [inline system/keyboardlayouts/hungarian.xml]: not sure if we have any 'standard' regarding this, but all those characters are already available in the symbol section.

Comment (Wolftronics-SBC) [reply system/keyboardlayouts/hungarian.xml]: yes there are a symbol section. I think so you can use a keyboard faster.

Comment (Wolftronics-SBC) [reply system/keyboardlayouts/hungarian.xml]: Updated hungarian.xml frequently used buttons add to keyboard layout... 
fe1550f  to cd91fd8

File: addons/skin.estuary/xml/DialogButtonMenu.xml (hunk 1 above)
Comment (ronie) [reply addons/skin.estuary/xml/DialogButtonMenu.xml]: ^ @Wolftronics-SBC  we can't merge this PR until this is fixed ;-)

**Comment by Razzeee** [comment]
This seems to be superseeded by https://github.com/xbmc/xbmc/pull/12860/files

**Comment by sakos** [comment]
This would fix [#17260](https://trac.kodi.tv/ticket/17260) ticket.

BTW, what about other skins? Will they all need change accordingly?

**Comment by Wolftronics-SBC** [comment]
key "ó" visible working no problem....

**Comment by ronie** [comment]
merged https://github.com/xbmc/xbmc/pull/12860
closing this one.
